plugins/kubernetes/app/models/watchers/deploy_watcher.rb

```diff
@@ -15,12 +15,11 @@ def initialize(release)
     end
 
     def watch
-      @release.release_docs.each do |release_doc|
-        subscribe("#{release_doc.replication_controller_name}", :handle_update)
-      end
+      subscribe("#{@release.build.project_name}", :handle_update)
     end
 
     def handle_update(topic, data)
+      info "Got Release Event: #{topic}"
```

Comment:
Yup, that was a temporary log while debugging the watcher. 'debug' would be sufficient if leaving this log in.